docs/index.rst

```diff
@@ -1,43 +1,41 @@
-.. scrapy-poet documentation master file, created by
-   sphinx-quickstart on Tue Apr 28 16:45:10 2020.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 =========================
 scrapy-poet documentation
 =========================
 
-scrapy-poet easily integrates Page Objects created using `web-poet`_ with
-Scrapy through the configuration of a dependency injection middleware.
+scrapy-poet allows to use `web-poet`_ Page Objects with Scrapy.
+
+web-poet_ defines a standard for writing reusable and portable
+extraction and crawling code; please check its docs_ to learn more.
 
-`web-poet`_ is used to make reusable Page Objects that separates
-extraction logic from crawling. They could be easily tested and distributed
-across different projects. Also, they could make use of different backends,
-for example, acquiring data from `Splash`_ and `AutoExtract`_ API.
+By using ``scrapy-poet`` you'll be organizing the spider code in a different
+way, which allows to separate extraction and crawling logic from the I/O,
+and from the Scrapy implementation details as well. It makes the code
```

Comment:
As markdown aggregates line-breaks in the same paragraph, maybe we could leave each phrase in it's own line, making it easier to review future changes.